.gitattributes

```diff
@@ -1,4 +1,6 @@
-*.pbxproj -diff linguist-generated
+*.pbxproj binary linguist-generated
+ios/Exponent.xcodeproj/project.pbxproj merge=mergepbx -text diff linguist-generated
+apps/bare-expo/ios/BareExpo.xcodeproj/project.pbxproj merge=mergepbx -text diff linguist-generated
```

Comment:
These lines results in being able to see diff for only two specified `.pbxproj` files (every else `.pbxproj` is treated as `binary` (`-diff -text`):
```
diff --git a/apps/bare-expo/ios/BareExpo.xcodeproj/project.pbxproj b/apps/bare-expo/ios/BareExpo.xcodeproj/project.pbxproj
index af29fdbeed..664369f871 100644
--- a/apps/bare-expo/ios/BareExpo.xcodeproj/project.pbxproj
+++ b/apps/bare-expo/ios/BareExpo.xcodeproj/project.pbxproj
@@ -6,8 +6,6 @@
        objectVersion = 46;
        objects = {
-
-
 /* Begin PBXBuildFile section */
                00E356F31AD99517003FC87E /* BareExpoTests.m in Sources */ = {isa = PBXBuildFile; fileRef = 00E356F21AD99517003FC87E /* BareExpoTests.m */; };
                13B07FBC1A68108700A75B9A /* AppDelegate.m in Sources */ = {isa = PBXBuildFile; fileRef = 13B07FB01A68108700A75B9A /* AppDelegate.m */; };

diff --git a/apps/bare-expo/ios/Pods/Pods.xcodeproj/project.pbxproj b/apps/bare-expo/ios/Pods/Pods.xcodeproj/project.pbxproj
index 85e3e54d20..1d3bbf20b5 100644
Binary files a/apps/bare-expo/ios/Pods/Pods.xcodeproj/project.pbxproj and b/apps/bare-expo/ios/Pods/Pods.xcodeproj/project.pbxproj differ

diff --git a/ios/Exponent.xcodeproj/project.pbxproj b/ios/Exponent.xcodeproj/project.pbxproj
index 5b42fb5434..bc57b61db4 100644
--- a/ios/Exponent.xcodeproj/project.pbxproj
+++ b/ios/Exponent.xcodeproj/project.pbxproj
@@ -1,6 +1,8 @@
 // !$*UTF8*$!
 {
        archiveVersion = 1;
+
+
        classes = {
        };
        objectVersion = 46;
```